doc/ref_model/chapters/appendix-a.md

```diff
@@ -1,73 +1,50 @@
 [<< Back](../../ref_model)
-# Appendix A - Guidelines For VNF Vendors
+# Appendix A - Guidelines
 <p align="right"><img src="../figures/bogo_ifo.png" alt="scope" title="Scope" width="35%"/></p>
 
 ## Table of Contents
-* [A.1 Goals](#A.1)
-* [A.2 Intro and Terminology](#A.2)
-* [A.3 VNF Evolution Phases](#A.3)
-* [A.4 Links](#A.3)
-* [A.5 Hardware-Dependent Coding Policies](#A.4)
-* [A.6 Miscellaneous](#A.5)
+* [A.1 Guidelines for VNF Vendors](#A.1)
+* [A.2 Guidelines for CNF Vendors](#A.2)
```

Comment:
I am not sure there should be different guidelines for VNF or CNF vendors. Also a CNF is a VNF.

Reply:
As above, I disagree that a CNF is a VNF.  I think it is much more useful to consider them as separate epochs of network software, if you like, in the same way that VNF is a step up from PNF in terms of software architecture.

Reply:
The architecture of CNFs and the way they are developed, delivered and operated must be different from VNFs - that surely means they are different.

Reply:
@tomkivlin I think you would agree that VNFs (targeted for VMs) can also be developed using cloud native principles -- microservices, stateless, ... -- and processes such as "immutability" used (although expensive in time to do so). The real difference being in the deployment and orchestration of the NFs

Reply:
@pgoyal01 I agree CNFs can consist of VMs, sure - and bare metal components,  etc.. I disagree that VNFs can be developed using cloud native principles. A network function developed using cloud native principles is surely a CNF?

Reply:
@tomkivlin My understanding of our position is that ultimately we want to see cloud native principles baked into both VNFs and CNFs, but all else being equal, an NF on a VM is referred to as a "VNF" and an NF in a container is referred to as a "CNF", as they have different implications.

Reply:
@markshostak ah ok, this is where I differ, perhaps I'm alone in CNTT but I want to make sure my position is clear, even if we don't agree.  
For me, a CNF could consist of containers, VMs, physical machines, serverless functions, etc. Cloud native is more about how software is architected, developed and LCM'd than what it runs on.  Hence why RA2 scope includes ability for Kubernetes to orchestrate VMs. And I would also not expect a VNF to adopt cloud native principles - if that were the case, what's the difference between VNF and CNF?

Reply:
@tomkivlin key differences: declarative specifications (hw, sw, etc.) and workloads deployed in clusters orchestrated by kubernetes. In this scenario CNF is "conatinerised" NFs that are "fully cloud-native." 

VNFs do not require being "cloud native" but doesn't preclude it.

Reply:
I would then say that a VNF that is cloud native can be deployed as part of a NFV NS using NFVO's Os-Ma-nfvo reference point.
My question is what would we do if we had a cloud native NF that is not a VNF, so it currently cannot be part of a NFV NS and thus cannot be managed by the Os-Ma-nfvo reference point?